src/common/RunParams.cpp

```diff
@@ -316,6 +318,33 @@ void RunParams::parseCommandLineOptions(int argc, char** argv)
         input_state = BadInput;
       }
 
+    } else if ( opt == std::string("--align") ||
```

Comment:
Typically, we do a long form with '--' prefix and a short form with '-' prefix. Is that the intent here; i.e., '---align' is the short form? If so, I think it should have only one dash.

Also, is 'data' alignment a better name than 'memory' alignment?

Reply:
that's reasonable, I'll change that